DESCRIPTION

```diff
@@ -20,15 +24,16 @@ Imports:
     magrittr,
     plotly,
     progress,
-    rlang,
+    rlang (>= 1.1.0),
     R6,
-    purrr,
+    purrr (>= 1.0.0),
     stringr
 Suggests: 
     covr,
     knitr,
     mockery,
     rmarkdown,
-    testthat (>= 3.0.0)
+    testthat (>= 3.0.0),
+    withr
 Config/testthat/edition: 3
 VignetteBuilder: knitr
```

Comment:
It's worth to prepare some vignettes covering:
- full sample workflow (setup and getting results) - this can be also partially covered in Readme.
- configuring minimal token setup.

Reply:
Vignettes will be prepared in #37